generators/new-wp-plugin/index.js

```diff
@@ -92,11 +91,33 @@ module.exports = generators.Base.extend({
       regex: 'lean-p',
       replacement: this.name
     }, settings) );
+    console.log( chalk.green('Completed!\n') );
   },
 
   install: function() {
     process.chdir( this.PLUGIN_FOLDER );
 
-    this.spawnCommandSync('composer', ['update']);
+    if ( fs.existsSync('./vendor') ) {
+      console.log( chalk.red('Composer has alredy been installed\n') );
+    } else {
+      this.spawnCommandSync('composer', ['install']);
+      this.spawnCommandSync('composer', ['update']);
+    }
+  },
+
+  end: function(){
```

Comment:
Yes agreed.

> On Jun 1, 2016, at 8:26 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In generators/new-wp-plugin/index.js <https://github.com/moxie-lean/generators/pull/31#discussion_r65358853>:
> 
> >    },
> >  
> >    install: function() {
> >      process.chdir( this.PLUGIN_FOLDER );
> >  
> > -    this.spawnCommandSync('composer', ['update']);
> > +    if ( fs.existsSync('./vendor') ) {
> > +      console.log( chalk.red('Composer has alredy been installed\n') );
> > +    } else {
> > +      this.spawnCommandSync('composer', ['install']);
> > +      this.spawnCommandSync('composer', ['update']);
> > +    }
> > +  },
> > +
> > +  end: function(){
> Why is this doing? If we're just creating a plugin then the .github folder should stay in the plugin route. The .github folder for the project can be created by the new-wp-project generator. Same for .travis.yml (which is already done by new-wp-project).
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/moxie-lean/generators/pull/31/files/71c9ab02e1744c55d6781a72d662c355f667938d#r65358853>, or mute the thread <https://github.com/notifications/unsubscribe/ADvVibET4q4f648v68F4Tw2vO9M6cU2Yks5qHYh6gaJpZM4InFiZ>.
>